pkg/deploy/generator/resources.go

```diff
@@ -686,7 +687,7 @@ func (g *generator) vmss() *arm.Resource {
 	}
 
 	parts = append(parts,
-		"'MDMIMAGE=''"+version.MdmImage("")+"''\n'",
+		"'MDMIMAGE=''"+version.MdmImage(acrDomainSuffix)+"''\n'",
```

Comment:
These changes to this file can be dropped.  This is an obscure hack; we splice the correct ACR domain name on the front of MDMIMAGE before we use it (`MDMIMAGE="${RPIMAGE%%/*}/${MDMIMAGE##*/}"`) to avoid needing another ARM template parameter.

Reply:
I would also have to revert the changes made to `version.MdmImage` in order to revert these changes here. The hardcoded domain was removed from that function and the other two next to it.